paper/main.tex

```diff
@@ -1259,7 +1237,7 @@ \subsubsection*{Functional neuroimaging data collected during story
 condition listened to the words in a randomized order), or lay resting
 with their eyes open in the scanner (rest condition; 36
 participants).  Full neuroimaging details may be found in the original
-paper for which the data were collected~\citep{SimoEtal16}.
+paper for which the data were collected~\citep{SimoEtal16}. Procedures were approved by the Princeton University Committee on Activities Involving Human Subjects, and by Western Institutional Review Board (Puyallup, WA). All subjects were native English speakers with normal hearing and provided written informed consent.
```

Comment:
Change to:

Procedures were approved by the Princeton University Committee on Activities Involving Human Subjects, and by *the* Western Institutional Review Board (Puyallup, WA). All subjects were native English speakers with normal hearing and provided written informed consent.

(i.e., at "the" before Western Institutional Review Board)-- I assume you copied this line from Janice's paper?

Reply:
Sorry... I mean Erez's paper

Reply:
I did and I changed it to add 'the'